pkg/util/deployment/deployment.go

```diff
@@ -33,6 +34,10 @@ import (
 const (
 	// The revision annotation of a deployment's replica sets which records its rollout sequence
 	RevisionAnnotation = "deployment.kubernetes.io/revision"
+	// SurgePodsAnnotation is used for determining scaling events vs rollouts in the deployment
+	// controller. Still, this will be inaccurate if a new desired replica count and/or maxSurge
```

Comment:
I need to update this comment. Not true, since I updated the strategy hooks to account for it.